test/unit/mdc-select/mdc-select.test.js

```diff
@@ -41,10 +41,16 @@ class FakeMenu {
 function getFixture() {
   return bel`
     <div class="mdc-select" role="listbox" tabindex="0">
-      <span class="mdc-select__selected-text">Pick a food group</span>
-      <div class="mdc-select__menu mdc-simple-menu">
-        <ul class="mdc-simple-menu__items">
-          <li class="mdc-list-item" role="option" tabindex="0">An option</li>
+      <div class="mdc-select__surface">
```

Comment:
I thought you were talking about needing to nest this fixture within another div to make this work, but I don't see an additional parent div from the last commit, I just see added indentation. What happened?

Reply:
I read the docs... hahaha

You can operate on the fixture as a whole as if it were any other dom node, so instead of trying to select on `mdc-select` in the test I can just do `fixture.setAttribute('style', 'whatever')`